docs/deprecations.md

```diff
@@ -26,4 +26,4 @@ being deprecated.
 | [`v1alpha1.Run` is deprecated](https://github.com/tektoncd/community/blob/main/teps/0114-custom-tasks-beta.md)                                                                                                               | v0.43.0                                                              | Alpha                                                                                                  | April 10, 2023 or v0.47.0           |
 | [ClusterTask is deprecated](https://github.com/tektoncd/pipeline/issues/4476)                                                                                                                                                | v0.41.0                                                              | Beta                                                                                                   | July 13, 2023                       |
 | [`pipelineRef.bundle` and `taskRef.bundle` are deprecated](https://github.com/tektoncd/pipeline/issues/5514)                                                                                                                 | v0.41.0                                                              | Alpha                                                                                                  | July 13, 2023                       |
-| [`taskrun.status.cloudEvents` are deprecated](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md))                                                                                     | v0.44.0                                                              | Alpha                                                                                                  | Oct 11, 2023                        |
+| [`taskrun.status.cloudEvents` is deprecated along with PipelineResources](https://github.com/tektoncd/community/blob/main/teps/0074-deprecate-pipelineresources.md)                                                                                     | v0.44.0                                                              | Alpha                                                                                                  | Oct 11, 2023                        |
```

Comment:
suggest removing this line and updating the PipelienResources line to "`PipelineResources` are deprecated, along with the components of the API that rely on them. See [new section] for more info.

Then in the new section, you can list out:
- the `resources` fields of Task, TaskRun, Pipeline, and PipelineRun
- images used by PipelineResources
- cloud_event_count (and any other metrics used by resources)
- taskRun.status.cloudEvents

Reply:
Thanks Lee. Updated.

*Also regarding that we might want to keep the deprecation.md brief, I didn't add details under the list but I can do so if preferred:)

Reply:
I think it's worth adding some details! I'd include a summary sentence explaining that these are the features deprecated as part of the PipelineResource deprecation, specify which metrics and images are deprecated, and update the section title since not everything being deprecated is an API field.